### PR TITLE
Monitor status url of only active proxies

### PR DESCRIPTION
**Pull request by rafaeljesus**
Monitor health check endpoints only of active proxies

Ref: https://github.com/hellofresh/janus-dashboard/issues/203

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+## Added
+- Monitor health check endpoints only of active proxies. Reported on #203
```
Comment (vgarvardt) [inline CHANGELOG.md]: I think it is fix, not an addition =)

File: pkg/api/mongodb_repository.go [hunk 2]
```diff
@@ -98,14 +98,20 @@ func (r *MongoRepository) Remove(name string) error {
...
+// FindValidAPIHealthChecks retrieves all active apis that has health check configured
 func (r *MongoRepository) FindValidAPIHealthChecks() ([]*Definition, error) {
-	result := []*Definition{}
 	session, coll := r.getSession()
 	defer session.Close()
 
-	err := coll.Find(bson.M{"health_check.url": bson.M{"$exists": true}}).All(&result)
-	if err != nil {
+	query := bson.M{
+		"active": false,
```
Comment (vgarvardt) [inline pkg/api/mongodb_repository.go]: `false`?

Comment (rafaeljesus) [reply pkg/api/mongodb_repository.go]: 🤦‍♂️

**Comment by rafaeljesus** [comment]
Integration tests are failing cos I've rebased with master, which is broken due this https://github.com/hellofresh/janus/pull/221

**Comment by italolelis** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/239/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/239/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway) [Approved with PullApprove](https://pullapprove.com/hellofresh/api-gateway/pull-request/239/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=hellofresh/api-gateway)

**Pull request merged**